<launch>

<arg name="obstacle_num" default="5"/>
<arg name="robot_r" default="0.4"/>
<arg name="obstacle_radius" default="0.25"/>
<arg name="goal_dis_exper" default="3"/>
<arg name="time_max_exper" default="30"/>


<node name="expriment_node_vs" pkg="expriment_node" type="expriment_node_vs" >
  <param name="/obstacle_num" value="$(arg obstacle_num)"/>
  <param name="/robot_r"           value="$(arg robot_r)"/>
  <param name="/obstacle_radius"           value="$(arg obstacle_radius)"/>
  <param name="/goal_dis_exper"           value="$(arg goal_dis_exper)"/>
  <param name="/time_max_exper"           value="$(arg time_max_exper)"/>
</node>

  </launch>